.vscode/launch.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Where did you get this code from? You shouldn't have had to add any code to .vscode...

Reply:
This was automatically added somehow, never touched this one.